templates/field/field--entity-reference-revisions.html.twig

```diff
@@ -1,59 +1,16 @@
 {#
 /**
  * @file
- * Theme override for a field.
- *
- * To override output, copy the "field.html.twig" from the templates directory
- * to your theme's directory and customize it, just like customizing other
- * Drupal templates such as page.html.twig or node.html.twig.
- *
- * Instead of overriding the theming for all fields, you can also just override
- * theming for a subset of fields using
- * @link themeable Theme hook suggestions. @endlink For example,
- * here are some theme hook suggestions that can be used for a field_foo field
- * on an article node type:
- * - field--node--field-foo--article.html.twig
- * - field--node--field-foo.html.twig
- * - field--node--article.html.twig
- * - field--field-foo.html.twig
- * - field--text-with-summary.html.twig
- * - field.html.twig
- *
- * Available variables:
```

Comment:
**Question:** Same as above, can devs still find this? Would be it relevant to include in this template?

Reply:
My thinking is the base template should have all the variables, I think it's excessive to duplicate the same comment doc blocks in the variants.